.pre-commit-config.yaml

```
@@ -3,13 +3,21 @@ repos:
     rev: v0.3.4
     hooks:
       - id: ruff
-        name: Run Ruff on Lib/test/
+        name: Run Ruff (check) on Doc/
```

Maybe

```suggestion
        name: Run Ruff (lint) on Doc/
```

I know the Ruff subcommand to lint things is `ruff check`. But the fact that the `ruff-format` hook is the only one with the `--check` argument is kinda confusing now 😆

---

Fair enough; renamed